tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestExampleGenerator.java

```diff
@@ -2076,4 +2079,27 @@ private String getResult(String starredPath, String attr) {
     public String sampleAttrAndValue(PathStarrer ps, final String separator, String value) {
         return ps.getAttributesString(separator) + "➔«" + value + "»";
     }
+
+    public void TestRefactoring() {
```

Comment:
OK, i see it's for the temporary test.

Reply:
Ouch. All locales X all resolved paths will be way too much. 

Actually what you probably want to do here is to limit it by coverage and pathheader. I'll get some sample code for it.

Reply:
this code (will need some setup) should show whether a path would even be shown to users or not.  If not, it should be skipped from the test:

```java
// adapted from STFactory.isValidSurveyToolVote

        PathHeader.Factory phf = PathHeader.getFactory(null); // do this once!


        private boolean isValidSurveyToolVote(String xpath) {
            PathHeader ph = phf.fromPath(xpath); // will need a 
 PathHeaderFactory
            if (ph == null) return false;
            if (ph.getSurveyToolStatus() == PathHeader.SurveyToolStatus.DEPRECATED) return false;
            if (ph.getSurveyToolStatus() == PathHeader.SurveyToolStatus.HIDE
                    || ph.getSurveyToolStatus() == PathHeader.SurveyToolStatus.READ_ONLY) {
                return false;
            }

            if (supplementalDataInfo.getCoverageValue(xpath, locale.getBaseName())
                    > org.unicode.cldr.util.Level.COMPREHENSIVE.getLevel()) {
                return false;
            }
            return true;
        }
```

Reply:
Got it. I'll revise to test over only the paths that'd be shown to users. Even over that reduced set of paths, and considering the extra check needed to determine if a path shown to users, I'm guessing the resolved files may still be too large to test over.